app/main/views/service_updates.py

```diff
@@ -13,6 +13,44 @@
 from . import get_template_data
 
 
+@main.route('/service-status-updates', methods=['GET'])
+@main.route('/service-status-updates/<day>', methods=['GET'])
+@main.route('/service-status-updates/<day>/page-<int:page>', methods=['GET'])
```

Comment:
I know I'm very late to this, but why are day and page not query parameters? That's what we use for services and it would also allow pagination across dates.